src/rest/connector/libs/xpresso/__init__.py

```diff
@@ -0,0 +1,7 @@
+# Enable abstraction using this directory name as the abstraction token
+try:
+    from genie import abstract
+    abstract.declare_token(__name__)
+except Exception as e:
+    import warnings
```

Comment:
Imports should always be at the top of the file

Reply:
all other implementations use the same __init__, pretty sure the actual import is part of the try/catch